README.md

```diff
@@ -182,6 +182,34 @@ defmodule MyModule do
 end
 ```
 
+You can also add type parameters:
+```elixir
+defmodule User do
+  use TypedStruct
+
+  typedstruct do
+    arg :state
```

Comment:
Nit: I would call it `parameter`:
- to match the style of `field` and `plugin`, which are not abbreviated
- to match the [documentation](https://hexdocs.pm/elixir/1.14/typespecs.html#user-defined-types), which talks about "parameterized types"